packages/buttons/src/elements/Button.tsx

```diff
@@ -77,8 +81,13 @@ Button.defaultProps = {
   size: 'medium'
 };
 
+/**
+ * @extends HTMLAttributes<HTMLElement>
+ */
 export interface IIconProps extends HTMLAttributes<HTMLElement> {
+  /** Determines if the button is rotated */
   isRotated?: boolean;
+  /** Adds children to the button object */
```

Comment:
Any `children` prop you encounter should be `/** @ignore */` as this is a standard React prop.

Reply:
Fixed in latest update